db/migrations/mysql/000013_create_schemes.down.sql

```diff
@@ -0,0 +1 @@
+DROP TABLE IF EXISTS Schemes;
```

Comment:
Are we not reversing the alters before the drop anymore? Happy with it, just to acknowledge the change

Reply:
I am 0/5 with this, but i figured what's the chance that a customer will have the main table dropped and not the indexes?